.travis.yml

```diff
@@ -35,6 +37,17 @@ script:
 jobs:
   allow_failures:
     - gemfile: gemfiles/spree_master.gemfile
+  exclude:
+    - rvm: 2.5.1
+      gemfile: gemfiles/spree_3_7.gemfile
```

Comment:
@aplegatt do we need exclude  `Spree 3.7` for `2.5` if require in gemspec: `'>= 2.3.3'` [source](https://github.com/spree/spree/blob/v3.7.13/spree.gemspec#L11 )

Reply:
See this comment: https://github.com/spree-contrib/spree_braintree_vzero/pull/226#discussion_r528830583